conjurapi/client.go

```diff
@@ -35,6 +35,8 @@ type Router interface {
 	RotateAPIKeyRequest(roleID string) (*http.Request, error)
 
 	CheckPermissionRequest(resourceID, privilege string) (*http.Request, error)
+	ResourceRequest(resourceID string) (*http.Request, error)
+	ResourcesRequest(filter *ResourceFilter) (*http.Request, error)
```

Comment:
it would probably make sense to order these Router interface definitions in an alphabetical order

Reply:
OK.